File: src/main/java/um/tesoreria/core/hexagonal/chequera/chequeraSerie/infrastructure/web/dto/SpoterDataResponse.java
package um.tesoreria.core.hexagonal.chequera.chequeraSerie.infrastructure.web.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SpoterDataResponse {

    private Boolean status = false;
    private String message = "";
    private Integer facultadId;
    private Integer tipoChequeraId;
    private Long chequeraSerieId;
    private ChequeraSerieDto chequeraSerie;
}

File: src/main/java/um/tesoreria/core/kotlin/model/dto/SpoterDataResponse.kt
package um.tesoreria.core.kotlin.model.dto

data class SpoterDataResponse(

	var status: Boolean = false,
	var message: String = "",
	var facultadId: Int? = null,
	var tipoChequeraId: Int? = null,
	var chequeraSerieId: Long? = null,
	var chequeraSerie: ChequeraSerieDto? = null

)